organization/roadmap.md

```diff
@@ -0,0 +1,70 @@
+---
+type: Resource
+---
+
+# Roadmap
+
+This roadmap serves as a model for how we're developing, growing and scaling our organization. It sets out the roles for both the staff and the members of the Foundation for Public Code.
+
+## Phase 0: get the organization founded
+
+Staff of the Foundation for Public Code:
+
+* Build team
+* Map potential members
+* Establish a legal entity
+
+This phase is over when: the organization has a legal entity
+
+## Phase 1: Developing founding membership
+
+Staff of the Foundation for Public Code:
+
+* Build a diverse founding member base
+* Develop business cases and case study material
+* Research and develop alpha codebase stewardship
+* Define the value proposition for members
+* Define what founding membership would entail
+
+Founding members of the Foundation for Public Code:
+
+* Become founding members
+* Take co-ownership over the organization
+* Organize the first conference for founding members and other interested stakeholders
+* Build organizational alignment within their organizations and with other members
+* Shape the governance and financial structure of the organization
+* Commit time, resources and codebases to support the development of the organization and its activities
+
+This phase is over when: when there is a member base willing to commit resources and finances for phase 2
+
+## Phase 2: Building a scalable organization
+
+Staff of the Foundation for Public Code:
+
+* Provide beta codebase stewardship (no long term guarantees)
+* Develop scalable operations and communications
+* Establish a clear and convincing value proposition, impact model, business cases, and case studies
+* Evaluate and refine activities together with members
+
+Members of the Foundation for Public Code:
+
+* Develop a scalable business and governance model
+* Develop a growth model for the organization
+
+This phase is over when: the members are willing to commit to stage 3 and scaling
+
+## Phase 3: Scale the organization
+
+Staff of the Foundation for Public Code:
+
+* Provide codebase stewardship at scale
+* Grow our member base aggressively
+* Engage partner and affiliated organizations
+
+Members of the Foundation for Public Code:
+
+* Govern the organization
+* Actively participate in codebase stewardship and other activities of the organization
+* Support the long term sustainability and growth of the organization for the benefit of all members
+
+This phase is over when: when the [mission](../organization/mission.md) is fulfilled
```

Comment:
The file `mission.md` is in the same folder.

```suggestion
This phase is over when: when the [mission](mission.md) is fulfilled
```

For more information, see: https://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/